ansible/roles/ocp4-workload-rhtr-iot-managed/tasks/workload.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
For readability, please add a line between each task to keep standards throughout as much as possible.